src/modules/common/components/__tests__/FailureSwitcher.spec.js

```diff
@@ -29,16 +28,30 @@ describe('FailureSwitcher', () => {
       const error = new ContentNotFoundError({ type: 'event', id: '1234', language, city })
       expect(FailureSwitcher.renderContentNotFoundComponent(error)).toMatchSnapshot()
     })
+
+    it('should render a poi not found failure and match snapshot', () => {
+      const error = new ContentNotFoundError({ type: 'poi', id: 'poiId', language, city })
+      expect(FailureSwitcher.renderContentNotFoundComponent(error)).toMatchSnapshot()
```

Comment:
Do you think it makes sense to just assert on this method instead of really rendering the FailureSwitcher?

Reply:
Still relevant

Reply:
Hmm. It seems github didn't post my answer. I think it makes sense, as the Failure.spec.js should make sure that rendering of the Failure works. FailureSwitcher should just make sure that the right arguments are passed, don't you think?